docs/content/user-guides/cert-manager.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Traefik Cert Manager Integration"
+description: "Learn how to configure Traefik Proxy to use Cert Manager for certificate on your routers. Read the technical documentation."
```

Comment:
```suggestion
description: "Learn how to configure Traefik Proxy to use Cert Manager for certificate for your routers. Read the technical documentation."
```

Certificates are not tied to routes, the will be added to the cert store, and available then for any route, so I think we should avoid reinforcing this common misconception.

Reply:
That's right. :+1:  Thanks !